docs/blog/2018-08-30-gatsby-values/index.md

```diff
@@ -0,0 +1,167 @@
+
+Once we had all the anonymous ideas collected, Sam condensed them into common themes.
+
+### Step 3: Discuss the themes in person
+
+While we were all together, Sam went through the themes with all of us and we wrote them out on a whiteboard. This helped all of us get back into the context of what we’d written down before, and what the team collectively values.
+
+<figure>
+  <img alt="Kurt holding a sticky note with “always be open to change” written on it." src="./images/kurt-sticky-note.jpg" />
+  <figcaption>
+    Kurt writing down really good ideas on a sticky note. Photo:
+    <a href="https://www.houseofschaab.photography/">
+      Danielle Schaab
+    </a>
+  </figcaption>
+</figure>
+
+### Step 4: Capture additional ideas on sticky notes
+
+Once we all had context, we distributed sticky notes and markers, then set a timer for 15 minutes to allow everyone to add any additional ideas, values, or themes that weren’t covered in the original document.
+
+### Step 5: Group similar ideas and identify the core values
+
+With everyone’s ideas on the board, we started grouping. [Marisa](https://twitter.com/marisamorby) and I looked for common threads in the ideas on the whiteboard and started to condense them down into a set of core values.
+
+<figure>
+  <img alt="Kyle, Linda, and Marisa adding and grouping ideas on the whiteboard." src="./images/values-whiteboard.jpg" />
+  <figcaption>
+    Kyle, Linda, and Marisa adding and grouping ideas on the whiteboard. Photo:
+    <a href="https://www.houseofschaab.photography/">
+      Danielle Schaab
+    </a>
+  </figcaption>
+</figure>
+
+As we wrote these out, we discussed them with the whole team to make sure we weren’t oversimplifying or losing the message behind anyone’s ideas.
+
+At the end of the exercise, we’d extracted 8 shared core values, all of which the team is fully aligned on and deeply committed to.
+
+<blockquote class="twitter-tweet" data-lang="en"><p lang="en" dir="ltr">What we&#39;re doing at <a href="https://twitter.com/gatsbyjs?ref_src=twsrc%5Etfw">@gatsbyjs</a> this week: writing down what kind of company we want to be. <a href="https://t.co/N0SRaJXILS">pic.twitter.com/N0SRaJXILS</a></p>&mdash; Jason Lengstorf (@jlengstorf) <a href="https://twitter.com/jlengstorf/status/1032319257298628608?ref_src=twsrc%5Etfw">August 22, 2018</a></blockquote>
+
+## What we learned — Gatsby’s core values
+
+With our themes identified, we needed to polish our notes up into clear company values. Here’s what we settled on.
+
+### Do the right thing when no one is looking
+
+Companies live or die by the trust they build. Gatsby is an open source product, and that means we need to earn and maintain trust from our internal team, our larger community of contributors (and the open source community at large), as well as from our clients and customers.
+
+To earn that trust, we need to be deeply committed to always doing the right thing _for the sake of doing the right thing_.
```

Comment:
This one is hazy for me still, because what is the "right" thing? Not that this value isn't great--it just seems a little vague still.

Reply:
I think this needs to stay general. The "right thing" means not cutting corners because "no one will ever know", or doing something wrong because there's no chance of getting caught. In general, it means, "I'm going to do the thing that I would want everyone else to do in this situation."

If we get too specific, we lose the spirit of it and end up with a list of dos and don'ts. This isn't meant to be a checklist or an enforceable rule; it's meant to be a question we ask ourselves before we make a decision. "Would I be willing to admit that I did this in public? Will I be proud of this choice? Would I be happy if someone in the community made the decision I'm about to make?"

Ultimately, the core value is integrity, like @jettig29 proposed at the Gathering. Does that make sense? Should this be clarified?